server/routes.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Main application routes
+ */
+
+'use strict';
+
+var errors = require('./components/errors');
+
+module.exports = function(app) {
+  console.log(app.get('appPath'));
+
+  // Insert routes below
+  // Delete this route for things
+  // app.use('/api/things', require('./api/thing'));
+  // app.use('/api/action', require('./api/action'));
+  // app.use('/api/event', require('./api/event'));
+  // app.use('/api/observation', require('./api/observation'));
+  // app.use('/api/user', require('./api/user')); 
+
+  // app.use('Server/auth', require('./auth'));
+
+  // All undefined asset or api routes should return a 404
+  app.route('/:url(api|auth|components|app|bower_components|assets)/*')
+   .get(errors[404]);
+
+  // All other routes should redirect to the index.html
+  app.route('/*')
+    .get(function(req, res) {
+      res.sendFile(app.get('appPath') + 'index.html');
```

Comment:
@dmsakamoto what do you think about instead of serving the index.html for an invalid URL path, we just redirect the user to /? That way, with the redirect, the URL in the browser will show the correct path of index.html.

So replace `res.sendFile(app.get('appPath') + 'index.html');` with `res.redirect('/');`